src/mongoose_cets_discovery.erl

```diff
@@ -33,12 +33,20 @@ supervisor_specs(#{backend := DiscoBackend, cluster_name := ClusterName} = Opts)
         backend_module => disco_backend_to_module(DiscoBackend),
         cluster_name => atom_to_binary(ClusterName),
         node_name_to_insert => atom_to_binary(node(), latin1),
+        node_ip_binary => get_node_ip_binary(),
         name => mongoose_cets_discovery, disco_file => DiscoFile},
     CetsDisco =
         {cets_discovery,
           {?MODULE, start_link, [DiscoOpts]},
           permanent, infinity, supervisor, [cets_discovery]},
-    [CetsDisco].
+    Addr =
+        {mongoose_node_address,
+          {mongoose_node_address, start_link, []},
+          permanent, infinity, supervisor, [mongoose_node_address]},
+    [CetsDisco, Addr].
```

Comment:
I think it would be better to use the new map syntax for supervisor child specs, it's been around for a while.